users/details/script.js

```diff
@@ -112,15 +112,35 @@ function generateUserData(userData) {
     type: 'div',
     classList: ['user-details-wrap'],
   });
-  const username = createElement({
+  const fullName = createElement({
     type: 'h2',
+    classList: ['user-details-fullname'],
+  });
+  fullName.appendChild(
+    createTextNode(
+      userData?.first_name + ' ' + userData?.last_name ?? userData.username,
```

Comment:
I think it will replace lastname with username if not found. check it once

Reply:
Not needed, it's create confusion

Reply:
Thanks for the changes